evi-flutter-example/lib/main.dart

```diff
@@ -137,8 +137,9 @@ class _MyHomePageState extends State<MyHomePage> {
   // define config here for recorder
   static const config = RecordConfig(
     encoder: AudioEncoder.pcm16bits,
-    bitRate:
-        48000 * 2 * 16, // 44100 samples per second * 2 channels (stereo) * 16 bits per sample
+    bitRate: 48000 *
+        2 *
+        16, // 44100 samples per second * 2 channels (stereo) * 16 bits per sample
```

Comment:
we can update the 44100 to be 48000 here in the comment

Reply:
Done!